src/components/Main/Navigation/use/toolBox.ts

```diff
@@ -8,7 +8,7 @@ interface Tool {
   iconName: string
   iconMdi?: true
   disabled?: boolean
-  onClick: (event: MouseEvent) => void
+  onClick: (event: MouseEvent) => void // clickイベント と mousedownイベント&&中央クリック で呼ばれる
```

Comment:
```suggestion
  /**
   * clickイベントと中央クリックでのmousedownイベントで呼ばれる
   */
  onClick: (event: MouseEvent) => void
```
ホバーしたときに見やすいようにこういう書き方にしてほしいです :pray:

Reply:
あ～JSのコメントの書き方全然わかってなかった…申し訳ねぇ～